docs/template_plugin/tests/functional/op_reference/logical_or.cpp

```diff
@@ -0,0 +1,80 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include <ie_core.hpp>
+#include <ie_ngraph_utils.hpp>
+#include <ngraph/ngraph.hpp>
+#include <shared_test_classes/base/layer_test_utils.hpp>
+#include <tuple>
+
+#include "base_reference_test.hpp"
+
+using namespace reference_tests;
+using namespace ngraph;
+using namespace InferenceEngine;
+
+struct LogicalOrParams {
+    template <class IT, class OT>
+    LogicalOrParams(const ngraph::PartialShape& input_shape1, const ngraph::PartialShape& input_shape2, const std::vector<IT>& iValues1,
+                    const std::vector<IT>& iValues2, const std::vector<OT>& oValues)
+        : pshape1(input_shape1),
+          pshape2(input_shape2),
+          inType(ngraph::element::boolean),
+          outType(ngraph::element::boolean),
+          inputData1(CreateBlob(ngraph::element::boolean, iValues1)),
+          inputData2(CreateBlob(ngraph::element::boolean, iValues2)),
+          refData(CreateBlob(ngraph::element::boolean, oValues)) {}
+    ngraph::PartialShape pshape1;
+    ngraph::PartialShape pshape2;
+    ngraph::element::Type inType;
+    ngraph::element::Type outType;
+    InferenceEngine::Blob::Ptr inputData1;
+    InferenceEngine::Blob::Ptr inputData2;
+    InferenceEngine::Blob::Ptr refData;
+};
+
+class ReferenceLogicalOrLayerTest : public testing::TestWithParam<LogicalOrParams>, public CommonReferenceTest {
```

Comment:
Consider grouping Logical Op test classes together (similar approach to what you've done to type prop tests), it would basically be the same thing that was already done for comparison ops.
https://github.com/openvinotoolkit/openvino/blob/45982fc30fbd3cd1c4c94c73882ac1da9baf6d8d/docs/template_plugin/tests/functional/op_reference/comparison.hpp#L33
https://github.com/openvinotoolkit/openvino/blob/45982fc30fbd3cd1c4c94c73882ac1da9baf6d8d/docs/template_plugin/tests/functional/op_reference/less.cpp#L21

Reply:
Done